.github/workflows/acvm-test-acvm-js.yml

```diff
@@ -72,7 +77,12 @@ jobs:
         uses: actions/download-artifact@v3
         with:
           name: acvm-js
-          path: ./acvm-repo/result
+          path: ./result
+      
+      - name: Move build artifacts
+        run: |
+          mv ./result/acvm_js/nodejs ./acvm-repo/acvm_js/nodejs
+          mv ./result/acvm_js/web ./acvm-repo/acvm_js/web
```

Comment:
Tests now require the build artifacts to be in `acvm-repo/acvm_js` rather than in a `result` directory so we move them there. We can handle this in a more natural way later.